src/facades/ReleaseCommunicationFacade.js

```diff
@@ -82,7 +82,15 @@ class ReleaseCommunication {
     }, []);
 
     if (nonSquashedPRs.length) {
-      const prNumbersByCommit = await Promise.all(nonSquashedPRs.map(async commit => searchIssuesByCommitHandler(commit)));
+      let prNumbersByCommit;
```

Comment:
Do you want to maybe instantiate this as an array to avoid potential errors when you use `prNumbersByCommit.forEach` further down the road?

Reply:
Yes!